server/hot-reloader.js

```diff
@@ -73,6 +78,38 @@ export default class HotReloader {
       return
     }
 
+    const handlePageBundleRequest = async (req, res, parsedUrl) => {
```

Comment:
What's this function is doing?
May be add an comment.

Reply:
Added 👍